packages/holocron/src/loadModule.node.js

```diff
@@ -88,7 +88,9 @@ export default async function loadModule(
       }
       nodeModule = requireFromString(moduleString, url);
     } catch (err) {
-      if (err.shouldBlockModuleReload !== false) {
+      if (process.env.NODE_ENV !== 'production' && err.shouldBlockModuleReload !== true) {
```

Comment:
checking for 'development' is always safer than checking for 'production', I also think this should just be:
 
```suggestion
      if (err.shouldBlockModuleReload !== false && process.env.NODE_ENV !== 'development') {
```
I don't think we care about logging the error as it will be bubbled up, just want to ensure that in development the module does not get added to the blocklist.

Reply:
I've updated ( 5233682 ) to check only for production and wrote some more notes on it here:

https://github.com/americanexpress/holocron/pull/31/files#r430669347